src/components/attribute-filter.tsx

```diff
@@ -83,10 +84,10 @@ export const AttributeFilter = () => {
               let filterValue;
               const attrFilter = frequencies[selectedFrequency].filters.find(f => f.attribute === attr.name);
               if (attrFilter) {
-                const filterAboveOrBelowMean = (attrFilter.operator === "aboveMean" || attrFilter?.operator === "belowMean");
+                const noValueFilter = (attrFilter.operator === "aboveMean" || attrFilter?.operator === "belowMean" || attrFilter?.operator === "all") ;
```

Comment:
The mix of `attrFilter.operator` and `attrFilter?.operator` here may be confusing for future developers.